tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -135,4 +139,17 @@ func testFilesCopied(t *testing.T) {
 		assert.Emptyf(t, stdout.String(), "Missing file: %s", fullPath)
 	}
 
+	// Check the SHA of kube.tar.gz downloaded
+	kubeTarPath := ansibleTempDir + "\\" + "kube.tar.gz"
+	command := fmt.Sprintf("certutil -hashfile %s SHA512", kubeTarPath)
+	stdout := new(bytes.Buffer)
+	_, err = client.Run(command, stdout, os.Stderr)
```

Comment:
If `command` returns an error, does that get reflected in `err`? I thought you would need to check `Stderr`.